lib/vmdb/global_methods.rb

```diff
@@ -67,6 +67,23 @@ def format_timezone(time, timezone = Time.zone.name, ftype = "view")
       new_time
     end
 
+    # Create a time in a timezone, return in UTC
+    def create_time_in_utc(datetime, tz = nil)                # tz = nil means use user's session timzone
```

Comment:
@Fryguy : I was not able to give a sound advice on the placement of this. So discussion and advice on this topic is highly welcome.